File: api/src/main/java/br/cefet/tutorParticular/model/Professor.java
package br.cefet.tutorParticular.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Professor {
    private int idProfessor;
    private String nome;
    private String cpf;
    private String email;
    private String senha;
    private String telefone;
    private String uf;
    private String cep;
    private String cidade;
    private String bairro;
    private String rua;
    private int numero;
    private int numMaxAlunos;
    private String apresentacao;
    private double mediaAvaliacao;
    private String foto;
}

File: projeto/src/app/model/professor.ts
export class Professor {
    idProfessor: number;
    nome: string;
    cpf: string;
    email: string;
    senha: string;
    telefone: string;
    uf: string;
    cep: string;
    cidade: string;
    bairro: string;
    rua: string;
    numero: number;
    numMaxAlunos: number;
    apresentacao: string;
    mediaAvaliacao: number;
    foto: string;

    constructor(){
        this.idProfessor = 0;
        this.nome = "";
        this.cpf = "";
        this.email = "";
        this.senha = "";
        this.telefone = "";
        this.uf = "";
        this.cep = "";
        this.cidade = "";
        this.bairro = "";
        this.rua = "";
        this.numero = null;
        this.numMaxAlunos = null;
        this.apresentacao = "";
        this.foto = null;
    }
}
